html/elements/meta.json

```diff
@@ -124,6 +124,469 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "height": {
+            "__compat": {
+              "description": "content=\"height=[value]\"",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Viewport_meta_tag#height",
+              "spec_url": "https://drafts.csswg.org/css-viewport/#viewport-meta",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": "18"
+                },
```

Comment:
How did you determine this given that the Chrome Android 18 release notes don't seem to mention these features?